pkgs/applications/editors/vim/plugins/non-generated/jsonfly-nvim/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitea,
+  vimUtils,
+  nix-update-script,
+}:
+vimUtils.buildVimPlugin {
+  pname = "jsonfly.nvim";
+  version = "0-unstable-2025-06-07";
+
+  src = fetchFromGitea {
+    domain = "git.myzel394.app";
+    owner = "Myzel394";
+    repo = "jsonfly.nvim";
+    rev = "db4394d856059d99d82ea2c75d033721e9dcb1fc";
+    hash = "sha256-PmYm+vZ0XONoHUo08haBozbXRpN+/LAlr6Fyg7anTNw=";
+  };
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "❴🦋❵ Fly through your JSON / XML / YAML files with ease. Search ✨ blazingly fast ✨ for keys via Telescope.";
```

Comment:
Please, get rid of emojis in the package descriptions.
Also, remove the final period (https://nixos.org/manual/nixpkgs/unstable/#var-meta-description).